### DISCUSSION TITLE
GPT 4 Turbo and Assistants API

### DISCUSSION THREAD

**Discussion by dspivak01** [betalgo/openai#409] [General]
Hey everyone,

I recently discovered this Open AI wrapper, and I've loved leveraging it to interact with the Open AI API. Recently, OpenAi released their new updates with Gpt 4 Turbo and the Assistants API. I believe that the application I'm building will significantly benefit from these new features, so I am curious how long it normally takes after updates for this wrapper to reflect the new features. I'm going to go ahead and switch back to using an HTTP client for requests to the OpenAI API for now, but I hope to switch back to using Betalgo once it has been updated to leverage the assistants API. If anyone can provide some insight on turn around time, that would be extremely helpful!

Thanks,
Dan

**Comment by richardtallent-erm** [comment]
I came here to ask specifically about whether anyone is already working on the lower-hanging fruit -- the new Seed and JSON output parameters, and a const for the turbo model name. Unfortunately, I can't contribute to that this weekend, but possibly next weekend if no one else is taking it on.

**Reply by ChaseIngersol** [reply]
@richardtallent-erm Yes, working on that [here](https://github.com/betalgo/openai/pull/406).
